src/configuration.py

```diff
@@ -74,6 +74,12 @@
     "{hugger} couldn't figure out how to hug themselves"
 ]
 
+WORDS_CENSORED = [
+    "horny",
```

Comment:
Possibly make this a regex to make it easier to stop bypasses

Reply:
Add time limit for regex validation to avoid bot freeze due to regex issues/bad regex

Reply:
or just test it and write valid regex